specification/security/_types/RoleDescriptor.ts

```diff
@@ -25,23 +25,61 @@ import { Metadata } from '@_types/common'
 import { OverloadOf } from '@spec_utils/behaviors'
 
 export class RoleDescriptor {
+  /**
+   * A list of cluster privileges. These privileges define the cluster level actions that API keys are able to execute.
+   */
   cluster?: string[]
-  /** @aliases index */
+  /**
+   * A list of indices permissions entries.
+   * @aliases index
+   */
   indices?: IndicesPrivileges[]
+  /**
+   * An object defining global privileges. A global privilege is a form of cluster privilege that is request-aware. Support for global privileges is currently limited to the management of application privileges. This field is optional.
```

Comment:
> ... This field is optional.

Not a big deal but IMO this text is unnecessary, since the "?" already indicates it's optional.

Reply:
Thanks! I missed those. I've removed them.